.prettierrc

```diff
@@ -1,5 +1,7 @@
 {
   "tabWidth": 2,
   "useTabs": false,
-  "trailingComma": "all"
+  "trailingComma": "all",
+  "editorconfig": true,
+  "printWidth": 160
```

Comment:
This is a styling change, not additional metadata, which is suppose to help users using some editors. Please remove that change.